app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
   after_initialize :set_defaults
+
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable,
+  devise :database_authenticatable,
+         :registerable,
+         :recoverable,
+         :rememberable,
+         :validatable,
```

Comment:
- [ ] Please kindly style your login and signup page 🙃 🙃 

Your design

<img width="402" alt="Screen Shot 2023-02-02 at 16 44 15" src="https://user-images.githubusercontent.com/74501523/216344720-fd2ad1d7-db45-4cd5-a484-d0646324b7c5.png">

Wireframe

<img width="402" alt="Screen Shot 2023-02-02 at 16 44 49" src="https://user-images.githubusercontent.com/74501523/216344812-21706c94-239f-4872-af23-f9e2ab75e44b.png">